recipes/csaps/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "csaps" %}
+{% set version = "0.11.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 977644db0aa1a10b9e99c7159ea7f70b064310b8fc4bfd8fc6d293b0bfc3c385
+
+build:
+  number: 0
```

Looks like this one can be noarch.
```suggestion
  noarch: python
  number: 0
```